core/src/test/java/org/apache/calcite/test/SqlOperatorUnparseTest.java

```diff
@@ -99,6 +99,11 @@ String rewrite(String sql) throws SqlParseException {
     }
   }
 
+  @Override @Disabled("Runtime error message differs after parsing and unparsing")
```

Comment:
Something is weird here, because the super function called doesn't have the same name. 
But since the test is disabled, it doesn't really matter.
This fixture was created to find bugs in unparsing, and it's possible that it does find such a bug. I will investigate, and if that's the case I will file an issue for it.

Reply:
@mihaibudiu  ooops, my bad, this looks like a copy-paste error (which does not have any impact due to the test being disabled), I meant to write:
```
@Override @Disabled("Runtime error message differs after parsing and unparsing")
void testBitAndFuncRuntimeFails() {
  super.testBitAndFuncRuntimeFails();
}
```
Feel free to correct this in the PR of [CALCITE-5990](https://issues.apache.org/jira/browse/CALCITE-5990).

The reason I disabled it was that the runtime error message contains the SQL query, and it seems that after the parse+unparse, the SQL query was not exactly the same as the original one (e.g. newlines were added), so the actual error was not matching the expected one.